gradle/versions.gradle

```diff
@@ -18,7 +18,7 @@ ext {
     vision_safety = vision
 
     mapbox_core = '1.3.0'
-    mapbox_telemetry = '4.5.1'
+    mapbox_telemetry = '4.4.1'
```

Comment:
Add note about the version downgrade and possibly link to the related issue in GH?